pipeline/processor/postprocessor.py

```diff
@@ -0,0 +1,634 @@
+import re
+from collections import Counter
+from collections import defaultdict
+from copy import deepcopy
+from typing import List, Dict, Optional
+
+import pandas as pd
+from english_words import english_words_set
+
+from pipeline.extract.document import Document, TextBlock
+
+
+class PostProcessor:
+    @staticmethod
+    def _minimal_bounding_box(coords: List[list]) -> list:
+        """
+        Return the minimally enclosing bounding box of bounding boxes.
+
+        Args: coords: A list of coordinates for each bounding box formatted [x1,y1,x2,y2] with the bottom left as the
+        origin.
+
+        Returns:
+            A list of coordinates for the minimally enclosing bounding box for all input bounding boxes.
+
+        """
+        x_min = min(coord[0][0] for coord in coords)
+        y_min = min(coord[1][1] for coord in coords)
+        x_max = max(coord[1][0] for coord in coords)
+        y_max = max(coord[3][1] for coord in coords)
+        return [[x_min, y_min], [x_max, y_min], [x_min, y_max], [x_max, y_max]]
+
+
+class HyphenationPostProcessor:
```

Comment:
This class relies on english words library to decide how to deal with hyphenation. Not robust to
other languages. Any ideas on how to deal with this?

Reply:
Maybe using a dataset accessible from `nltk.corpus` could be better, e.g. [wordnet](https://www.nltk.org/howto/wordnet.html)? NLTK also contains corpora in other languages we could extend to later, but the language issue isn't too big a deal for now as Adobe Extract is only English language.